lib/stat_tracker.rb

```diff
@@ -55,6 +45,21 @@ def count_of_teams
     teams.count
   end
 
+  # def find_team_by_id(id)
+  #   teams.find do |team|
+  #     team.team_id == id
+  #   end
+  # end
+
+  def find_team_by_id(array, id)
```

Comment:
Ah sorry, I'm re-reading and now catching you did that for the sake of flexibility - my bad. This is a creative solution! And I'm now seeing the challenge with a descriptive name for that parameter... let's think on it some more

Reply:
@rrabinovitch I actually just changed it back because last night I realized that this is just a 'find' method, not 'find_all', and there is only one collection you can pick a single team out of, and that's @teams.